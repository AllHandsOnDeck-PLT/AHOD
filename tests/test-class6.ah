main:
{
  Card card
  card = Card("R5", true, 5) 
  do PRINT(card.type)
  do PRINT(card.faceup)
  do PRINT(card.value)
}