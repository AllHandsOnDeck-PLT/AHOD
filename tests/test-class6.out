R5
1
5
R6
0
6
R7
1
7